api/lib/infrastructure/files/candidates-import/fill-candidates-import-sheet.js

```diff
@@ -11,27 +12,40 @@ const moment = require('moment');
 const _ = require('lodash');
 const FRANCE_COUNTRY_CODE = '99100';
 
-module.exports = async function fillCandidatesImportSheet(session) {
+module.exports = async function fillCandidatesImportSheet(session, certificationCenterHabilitations) {
   const stringifiedXml = await readOdsUtils.getContentXml({ odsFilePath: _getCandidatesImportTemplatePath() });
 
   const sessionData = SessionData.fromSession(session);
   const candidatesData = _getCandidatesData(session.certificationCandidates);
 
-  const updatedStringifiedXml = _updateXml(stringifiedXml, sessionData, candidatesData);
+  const updatedStringifiedXml = _updateXml(
+    stringifiedXml,
+    sessionData,
+    candidatesData,
+    certificationCenterHabilitations
+  );
 
   return writeOdsUtils.makeUpdatedOdsByContentXml({
     stringifiedXml: updatedStringifiedXml,
     odsFilePath: _getCandidatesImportTemplatePath(),
   });
 };
 
-function _updateXml(stringifiedXml, sessionData, candidatesData) {
-  const updatedStringifiedXml = sessionXmlService.getUpdatedXmlWithSessionData({
+function _updateXml(stringifiedXml, sessionData, candidatesData, certificationCenterHabilitations) {
+  let updatedStringifiedXml = sessionXmlService.getUpdatedXmlWithSessionData({
     stringifiedXml,
     sessionData,
     sessionTemplateValues: IMPORT_CANDIDATES_SESSION_TEMPLATE_VALUES,
   });
 
+  if (featureToggles.isComplementaryCertificationSubscriptionEnabled) {
+    if (!_.isEmpty(certificationCenterHabilitations)) {
+      updatedStringifiedXml = sessionXmlService.addAdditionalTableHeaders({
+        stringifiedXml: updatedStringifiedXml,
+        certificationCenterHabilitations,
+      });
+    }
+  }
```

Comment:
nitpick (pour avoir moins de diff lorsqu'on retirera le FT)
```suggestion
if (featureToggles.isComplementaryCertificationSubscriptionEnabled) {
  certificationCenterHabilitations.length = 0;
}
  if (!_.isEmpty(certificationCenterHabilitations)) {
    updatedStringifiedXml = sessionXmlService.addAdditionalTableHeaders({
      stringifiedXml: updatedStringifiedXml,
      certificationCenterHabilitations,
    });
  }

```